tests/plots/test_histogram.py

```
@@ -0,0 +1,341 @@
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import pytest
+
+from pyretailscience.plots.histogram import apply_range_clipping, plot
+from pyretailscience.style import graph_utils as gu
+
+
+# Fixture for mocking graph_utils functions
+@pytest.fixture()
```

Ruff

Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)